File: WebApp/Models/Price.cs
using WebApp.Persistence.Models;

namespace WebApp.Models
{
    public class Price
    {
        public int Id { get; set; }
        public int Cost { get; set; }
        public PassengerType PassengerType { get; set; }
        public TicketType TicketType { get; set; }
    }
}

File: Angular-app/src/app/pages/classes/price.ts
export class Price {
    Id: number;
    Cost: number;
    PassengerType: string;
    TicketType: string;

    constructor(price: Price) {
        this.Id = price.Id;
        this.Cost = price.Cost;
        this.PassengerType = price.PassengerType;
        this.TicketType = price.TicketType;
    }
}
